src/lib/store.js

```diff
@@ -4,6 +4,27 @@ import getMeta from './utils/meta';
 import {localStorage} from './utils/storage';
 import {isProd} from 'webdev_config';
 
+const initialParams = new URLSearchParams(window.location.search);
+
+let timeOffset = 0;
+if (initialParams.has('_now')) {
+  const overrides = {
```

Comment:
What do you think about something like:

```
?day1=2020-06-30T16:02&day2=2020-06-30T16:03
```

to override the days and make them 1 minute apart so we can watch the state transitions happen without needing hardcode timestamps into the js itself

Reply:
Um, I don't know. It's a lot of extra code for this. Having said that, testing with "_now" the way it is is also a pain because of our implicit day chooser: it's hard to see the transition.